blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueRun.java

```diff
@@ -193,8 +193,14 @@ public final String getEndTimeString(){
     @Navigable
     public abstract Object getLog();
 
+    /**
+     * Replays a pipeline. Sam SCM commit/revision should be used. in the new run.
+     *
+     * @return The queued item.
+     */
     @POST @TreeResponse @WebMethod(name = "replay")
```

Comment:
From REST API perspective: I see stop, replay verbs appearing in the API URL. Could we not use POST .../runs/:id with payload to indicate action (stop, replay)? Also it fits more of PUT than POST. PUT is considered idempotent, stop and replay both are in fact idempotent. 

```
POST /.../runs/1
{
  "action":"replay"
}
```